stdlib/public/Platform/SwiftGlibc.h.gyb

```diff
@@ -65,6 +65,7 @@ headers = [
   'spawn.h',
   'strings.h',
   'sys/event.h',
+  'sys/extattr.h',
```

Comment:
Is this header available universally? I think that this might need to be under a FreeBSD condition.

Reply:
The gyb expands '${header}' to an include guarded by `__has_include`, which test if the header can be included. similarly, the Linux specific `sys/inotify.h` header is also in the list.

```
#if __has_include(<${header}>)
#include <${header}>
#endif
```

Reply:
You're right, any headers added to this file are just ignored if they don't exist, as there are already some OpenBSD-specific headers.

Have you considered adding a FreeBSD overlay instead? You could take a look at the recent [Musl](https://github.com/swiftlang/swift/blob/20714b5fb940c79aca427c5ff574511b9c6891c3/stdlib/public/Platform/musl.modulemap), [WASI](https://github.com/swiftlang/swift/blob/e0131dc7dd0ad4999bcbb798b0e475b5a44ac5d4/stdlib/public/Platform/wasi-libc.modulemap), and [Android](https://github.com/swiftlang/swift/blob/22833fbc546147c46b058c62c72a08393e334bb5/stdlib/public/Platform/android.modulemap) overlays for examples.

I know that's more work, but the WASI one isn't so bad, and FreeBSD may not be either.

Reply:
That's definitely on my radar! I'm not sure yet if I have to bandwidth to include that change in this PR.

Reply:
We shoulda just called them all `import C`.